requirements.txt

```diff
@@ -1,4 +1,5 @@
 widgetastic.core~=0.21.4
 widgetastic.patternfly~=0.0.35
+widgetastic.patternfly~=0.1
```

Comment:
there seems the `4` missing - also please rebase to master and remove the pin here, as we learned the unfortunate way that those pin can be impediments to downstream like cfme as the version resolver of pip is rather unfortunate